### ISSUE TITLE
Fonts with only khmer subset cause 500 error

### ISSUE THREAD

**Issue by hexandcube** [varld/fontless#1]
Fonts that only have the Khmer subset cause internal server error 500 during deployment.

```
# This returns HTTP 500
https://google-webfonts-helper.herokuapp.com/api/fonts/suwannaphum?subsets=vietnamese,latin-ext,latin,greek-ext,greek,cyrillic-ext,cyrillic

# This does not
https://google-webfonts-helper.herokuapp.com/api/fonts/suwannaphum?subsets=vietnamese,latin-ext,latin,greek-ext,greek,cyrillic-ext,cyrillic,khmer,korean
```

Edit: The same issue is caused by fonts with only the Korean subset

**Comment by herber** [comment]
Thanks for the report. Do you have an idea about how we could fix this issue on fontless's side? 
It looks to me more like a problem with Google Webfonts Helper.

**Comment by hexandcube** [comment]
It looks like adding khmer and korean at the end of the URL on line 27 in /service/scripts/prepare-fonts.ts solves the problem.
![image](https://user-images.githubusercontent.com/41151156/103757017-75252700-5010-11eb-8c9e-794648af163f.png)

**Comment by herber** [comment]
Thanks for the report. I fixed it in #2 🎉
